test/remote/gateways/remote_payu_latam_test.rb

```diff
@@ -146,11 +146,11 @@ def test_successful_purchase_colombia
         zip: "01019-030",
         phone: "(11)756312633"
       ),
-      tx_tax: '3193',
-      tx_tax_return_base: '16806'
+      tax: "3193",
```

Comment:
Why the quote change?

Reply:
More so just making it consistent with the rest of test.